CRM/Utils/REST.php

```diff
@@ -176,6 +176,10 @@ public static function handle() {
 
     // Get the function name being called from the q parameter in the query string
     $q = $requestParams['q'] ?? NULL;
+    // On some user framworks e.g. WordPress the front end has its own query string so we use rq instead.
+    if (!empty($requestParams['rq'])) {
```

Comment:
@totten @kcristiano @christianwach I had to do this for the normal routing because the tests try to use a q param like `q=civicrm/contact/get` and this doesn't work with WordPress in non clean urls. I'm even not sure if there are any obvious docs that talk about the q= format

Reply:
@seamuslee001 There aren't any docs that I know of.